content/src/cloud/nabu-casa-account/log-out-all-devices.md

```diff
@@ -0,0 +1,16 @@
+---
+zendesk:
+  article_id: 26170426881693
+  name: Logging out all devices from Home Assistant Cloud
+  position: 24
+  labels: cloud
+---
+
+You can log out all devices that are currently logged in to Home Assistant Cloud. This includes Home Assistant, Google Home, Alexa, browsers, and the Home Assistant Companion App on any of your phones or tablets.
```

Comment:
You are not logged into your nabucasa account in the companion apps?

Reply:
Thank you!